sagemaker-pyspark-sdk/src/sagemaker_pyspark/SageMakerEstimator.py

```diff
@@ -116,6 +116,10 @@ def __init__(self, **kwargs):
     def _get_java_obj(self, **kwargs):
         raise NotImplementedError()
 
+    @property
+    def latestTrainingJob(self):
+        return self._call_java("latestTrainingJob")
```

Comment:
Can you add a unit test for this?

Reply:
Unit test added. But only initial None value is tested. The test for value change would be to much since it's hard to mock a scala fit call here.